### ISSUE TITLE
Styler: No text size on init

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#2645]
# Bug

**Describe the bug**
"Text size" box is empty when styler is opened.

**To Reproduce**
1. Go to 'Style layer'
2. Create or open some Text Symbolizer
4. See nothing in "text size" box

**Expected behavior**
Current size shall be displayed.

**Screenshots**
![obrazek](https://user-images.githubusercontent.com/5598693/152599260-f8647ce7-1951-405a-b262-9db1f41c1c7c.png)

**Additional context**
Some UX improvements can be also done on this field. Either changing it to slider like it is for e.g. width et al. Or perhaps adding some arrows on the side which would increment/decrement the number in the box by 1.
Like this I mean: ![obrazek](https://i.stack.imgur.com/7wJVC.png) or like this: ![obrazek](https://i0.wp.com/codemyui.com/wp-content/uploads/2016/11/custom-input-field.gif?fit=880%2C440&ssl=1)
But that could be another story for another issue and another release 🙂

**Comment by raitisbe** [comment]
I could only replicate when deleting the previous SLD rules and creating a new Rule for clusters in https://github.com/hslayers/hslayers-ng/pull/2643/commits/9c8a5d597f775103e04c80f41a588f2c40466d8a. When adding Text symbolizers text size is set to 12 for me.

**Comment by jmacura** [comment]
> I could only replicate when deleting the previous SLD rules and creating a new Rule for clusters in [9c8a5d5](https://github.com/hslayers/hslayers-ng/commit/9c8a5d597f775103e04c80f41a588f2c40466d8a). When adding Text symbolizers text size is set to 12 for me.

I was experiencing it when opening the styler for "Clusters without SLD". It seems to be OK now after #2643 .
